source/collections/VectorList.ooc

```diff
@@ -119,9 +124,12 @@ VectorList: class <T> {
 			result add(this[indices[i]])
 		result
 	}
-	getSlice: func (range: Range) -> This<T> {
-		result := This<T> new(range count)
-		this getSliceInto(range, result&)
+	getSlice: func (start, end: Int) -> This<T> {
```

Comment:
This is where I told you to remove the return statement, by mistake. Sorry about that...